apps/src/aichat/views/ChatWarningModal.story.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ChatWarningModal, {ChatWarningModalProps} from './ChatWarningModal';
+import {Meta, Story} from '@storybook/react';
+
+export default {
+  title: 'Edubot/ChatWarningModal',
+  component: ChatWarningModal,
+} as Meta;
+
+//
+// TEMPLATE
+//
+// This is needed to fix children type error (passing string instead of React.ReactNode type)
+// eslint-disable-next-line
+const SingleTemplate:Story<ChatWarningModalProps> = (args) => <ChatWarningModal {...args} />;
```

Comment:
Thanks for adding this!

Reply:
When I used literal strings, I was able to view the storybook!

<img width="1041" alt="Screen Shot 2023-08-15 at 10 35 58 AM" src="https://github.com/code-dot-org/code-dot-org/assets/107423305/760729a2-9b25-4ba0-9dca-a645aec04ab5">